money/money.py

```diff
@@ -30,127 +34,123 @@ def currency(self) -> Currency:
         return self._currency
 
     @classmethod
-    def from_sub_units(cls, sub_units: int, currency: Currency=Currency.USD):
+    def from_sub_units(cls, sub_units: int, currency: Currency = Currency.USD):
         """Creates a Money instance from sub-units."""
-        sub_units_per_unit = CurrencyHelper.sub_unit_for_currency(currency)
```

Comment:
This is a concrete change. Removed `CurrencyHelper`. Instead the field is defined for Currency Enum